Source/monster.h

```diff
@@ -300,6 +300,11 @@ void PrintMonstHistory(int mt);
 void PrintUniqueHistory();
 void PlayEffect(Monster &monster, int mode);
 void MissToMonst(Missile &missile, Point position);
+
+/**
+ * @brief Check that the given tile is not currently blocked
+ * @brief Check that the given tile is available to the monster

```

Comment:
There must be only one `@brief` annotation per doxygen comment. This is combining the comments for two different functions, you'll need to restore the `IsTileAvailable(Point)` comment and leave the `IsTileAvailable(Monster, Point)` version here.

Reply:
okay